### ISSUE TITLE
Alt Text doesn't display in its entirety depending on screen size

### ISSUE THREAD

**Issue by SpriterGors** [bluesky-social/social-app#5549]
### Steps to Reproduce

In certain cases where the Alt Text has many line breaks, the screen cannot accomodate it and will not display either the image or the Alt Text in its entirety.

How to reproduce:
1. Post a picture with a "tall" Alt Text (i.e with many line breaks);
2. Click on the picture to see its Alt Text.

### Attachments

Screenshots:
1- How the post appears in a 1366x768 screen:
![image](https://github.com/user-attachments/assets/3be9d7bc-6394-4e92-b42d-9ee3457c7018)

2- How it appears after clicking on the Alt Text. Note that the Alt Text is cropped and I cannot read the rest.
![image](https://github.com/user-attachments/assets/bf9ea024-41c2-44e6-95ca-1ef3dc6e853f)

3- How it appears in a 1920x1080 screen (it displays as expected):
![image](https://github.com/user-attachments/assets/4a5df51c-e20c-41cd-a8a3-efee3544f776)

### What platform(s) does this occur on?

Web (Desktop)

### Device Info

Windows 10 Pro 

### What version of the app are you using?

Build version: 1.91.1; Bundle info: 49ab486 (prod); Bundle date: 24092017; Platform: web

### Additional Information

This seems to do with screen size on my 1366x768 monitor not being enough to accomodate all info. However I suggest setting some kind of limit on how much the image can shrink so that I can still see it, and allow scrolling the Alt Text so I can read everything.
